src/group_theory/perm/list.lean

```diff
@@ -342,6 +347,109 @@ begin
       rw [hl, nat.mod_eq_of_lt hk', h1, ←nat.add_mod, nat.succ_add] } }
 end
 
+lemma form_perm_apply_mem_eq_self_iff (hl : nodup l) (x : α) (hx : x ∈ l) :
+  form_perm l x = x ↔ length l ≤ 1 :=
+begin
+  obtain ⟨k, hk, rfl⟩ := nth_le_of_mem hx,
+  rw [form_perm_apply_nth_le _ hl, hl.nth_le_inj_iff],
+  cases hn : l.length,
+  { exact absurd k.zero_le (hk.trans_le hn.le).not_le },
+  { rw hn at hk,
+    cases (nat.le_of_lt_succ hk).eq_or_lt with hk' hk',
+    { simp [←hk', nat.succ_le_succ_iff, eq_comm] },
+    { simpa [nat.mod_eq_of_lt (nat.succ_lt_succ hk'), nat.succ_lt_succ_iff]
+      using k.zero_le.trans_lt hk' } }
```

Comment:
Indenting to make it visually apparent the `using` line belongs to the `simpa` line.
```suggestion
        using k.zero_le.trans_lt hk' } }
```